### ISSUE TITLE
A couple items I think would be cool (link attached)

### ISSUE THREAD

**Issue by LelKappa** [studio-minus/ppg-bugs-and-requests#957]
**Feature description**
**Describe the requested feature. Is it a new spawnable item? Is it a new map? A new mechanic or system? How does it work?**

Various items I thought of while faded and playing ppg. Various weapons and a wagon that I think would be neat.

**What made you request this feature?** I love the game and I like to sketch so I had these ideas. **What is its purpose?** fo fun **Are you sure it doesn't already exist in some form?** Maybe in some form but I think a couple of them would be fun and interesting to play with in-game.

![IMG_2277](https://user-images.githubusercontent.com/18617653/159637213-c2887cc5-c898-49d5-a621-5d0dbe7bb25d.jpg)

**Comment by UniDuki** [comment]
Yeah you're not allowed to group multiple features in one request.
Maybe because zooi can't officially accept this feature without accepting all of them?
Cool drawings and cool ideas though ❤️
